<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>areena</class>
 <widget class="QWidget" name="areena">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>141</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>141</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tekstiboksi">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>101</width>
     <height>181</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:6pt;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="heroStats">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>71</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="vihuStats">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>61</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QProgressBar" name="xp_bar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>510</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>11</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
   <property name="format">
    <string>%v / %m xp</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="spell_box">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>181</width>
     <height>301</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="spell_layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="taioButton">
      <property name="text">
       <string>Taio</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>näkymättömyysviitta</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="heal_btn">
      <property name="text">
       <string>Spell heal</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="item_btn_class" name="pushButton">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="item_btn_class" name="warr_ultimate_btn">
      <property name="text">
       <string>ultimate punch</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="item_btn_class" name="warr_armup_btn">
      <property name="text">
       <string>armor up</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="item_btn_class" name="warr_findpot_btn">
      <property name="text">
       <string>find potion</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="item_btn_class" name="warr_trollstrike_btn">
      <property name="text">
       <string>troll srtike</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="item_btn_class" name="warr_p4life_btn">
      <property name="text">
       <string>pray4life</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="Hero_hp_bar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="format">
    <string>%v / %m</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="Enemy_hp_bar">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="format">
    <string>%v / %m</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="Hero_mp_bar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>127</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>63</red>
         <green>63</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>170</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>127</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>127</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>63</red>
         <green>63</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>170</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>127</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>127</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>63</red>
         <green>63</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>170</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="format">
    <string>%v / %m</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="Enemy_mp_bar">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="format">
    <string>%v / %m</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="Item_box">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="Hp_pot">
      <property name="text">
       <string>Health Potion</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Mp_pot">
      <property name="text">
       <string>Mana Potion</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="super_pot">
      <property name="text">
       <string>Super Potion</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="item_btn_class" name="lyoButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Lyö</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Item_storage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Items</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Spells_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Spells</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>item_btn_class</class>
   <extends>QPushButton</extends>
   <header location="global">item_btn_class.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
